### PR TITLE
Adds html content based on wireframe sketches.

### PR DESCRIPTION
**Pull request by katprohas**

### CODE REVIEW

File: artifacts.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document</title>
```
Comment (axamei) [inline artifacts.html]: I recommend changing the title.

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+
+<!-- Contact main content -->
+<h2><a id="contact">Contact Me!</a></h2>
+
+<p>(to be added - contact form)</p>
+<address>
+  Written by Katrina Prohaszka <br/>
+  Visit me at: <br/>
+  <a href=http://www.katsreadtoo.com>katsreadtoo.com</a> <br/>
```
Comment (axamei) [inline contact.html]: The value of the `href` attribute needs to be inside double quotes, e.g., `href="http://www.katsreadtoo.com"`

**Review by axamei — CHANGES_REQUESTED** [review]
In addition to the changes requested, I recommend adding a `<header>` and `<main>` element with the appropriate elements nested inside those tags. This will create more meaningful structure for your pages.

**Comment by katprohas** [comment]
@axamei Not sure if I should close the current pull request and open a new one for you, or leave this one open until you've had a chance to review the changes. Let me know what you would prefer. Thanks!

File: artifacts.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Artifacts Page</title>
+</head>
+<body>
+<!-- --------------------------------HEADER CONTENT--------------------------------------------------------------------------------- -->
+<header>
+<!-- Social Media Images -->
+
+<a href="https://www.facebook.com">
```
Comment (axamei) [inline artifacts.html]: These list of links to social media can be written as an unordered list.

File: artifacts.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+ Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Linkedin">Linkedin</a> from <a href="https://www.flaticon.com/"     title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"     title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a>
+</div>
+
+<br/>
+<br/>
+
+<!-- Banner for website with name and degree -->
+<h1 style="text-align: center"><u>KATRINA PROHASZKA</u></h1>
+<h3 style="text-align: center">MLIS in Information Management</h3>
+<div class="nav">
```
Comment (axamei) [inline artifacts.html]: instead of a div, use the `<nav>` element

File: artifacts.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+     <li><a href="artifacts.html">ARTIFACTS</a></li>
+     <li><a href="contact.html">CONTACT</a></li>
+   </ul>
+ </div>
+</div> <!-- end nav div -->
+</header> <!-- --------------------------------END HEADER CONTENT--------------------------------------------------------------------------------- -->
+
+<!-- --------------------------------MAIN CONTENT--------------------------------------------------------------------------------- -->
+<main>
+<br/>
```
Comment (axamei) [inline artifacts.html]: You can remove the line breaks and use CSS in the next assignment to control your spacing/visual presentation

**Review by axamei — CHANGES_REQUESTED** [review]

File: artifacts.html (hunk 3 above)
Comment (katprohas) [reply artifacts.html]: Changes have been made, please review and comment if there's anything additional to be done for this.

**Comment by axamei** [comment]
@katprohas I made some suggestions to the index.html file to clarify some change requests. Please review and apply to other pages. https://github.com/INF7420/prohaszka-project/pull/3/commits/433a4c1b999da64b895205aa764099d40353dc95. 

Run `git fetch` and `git pull` (or equivalent actions in GitKraken/Desktop app) to grab that commit I pushed into your html branch.

**Comment by katprohas** [comment]
I made the requested changes from previous, but have not yet done the assignment for tomorrow so those changes are not implemented in this push.

**Review by axamei — APPROVED** [review]

**Pull request merged**